pre_update.sh

```diff
@@ -1,5 +1,13 @@
 #!/bin/sh
 
+pgsql_id="$(id -u pgsql)"
+if [ "${pgsql_id}" != "769" ]; then
+	echo "Changing uid/gid of user/group pgsql"
+	pw groupmod -n pgsql -g 769
+	pw usermod -n pgsql -u 769 -g pgsql
+	find / -uid "${pgsql_id}" -exec chown -R pgsql:pgsql {} +
```

Comment:
Can we restrict to pgsql dirs and not whole root?
Also is there a find option to not traverse filesystems?